spec/lib/feature_management_spec.rb

```diff
@@ -132,28 +132,64 @@
     end
   end
 
-  describe '.fake_banner_mode?' do
-    context 'when in the production environment: secure.login.gov, idp.staging.login.gov' do
-      it 'does not display the fake banner' do
-        allow(LoginGov::Hostdata).to receive(:domain).and_return('login.gov')
-        allow(Rails.env).to receive(:production?).and_return(true)
-        expect(FeatureManagement.fake_banner_mode?).to eq(false)
+  describe '.show_demo_banner?' do
+    subject(:show_demo_banner?) { FeatureManagement.show_demo_banner? }
+
+    context 'in local development' do
+      # before { expect(LoginGov::Hostdata).to receive(:in_datacenter?).and_return(false) }
```

Comment:
can this be removed?

Reply:
whoopsidoodle, yes

Reply:
Fixed in 53f9477e8c793e5fb38a4e237d2ceebec8ad7f76